COLLABORATOR_GUIDE.md

```diff
@@ -87,6 +86,18 @@ The TC should serve as the final arbiter where required.
 
 ## Landing Pull Requests
 
+Pull requests should be landed with Jenkins as documented [here]
+(https://github.com/nodejs/node/wiki/Merging-pull-requests-with-Jenkins). 
+
+#### Landing Pull Requests manually (for emergency only)
```

Comment:
I wanted to preserve the information but avoid misguiding the reader into think that the manual procedure should be followed other than if mandated by emergencies. Somehow making those smaller made me thing that the reader would not skip the paragraph at ## Landing Pull Requests